Demo/EarlGreyExample/EarlGreyExample.xcodeproj/project.pbxproj

```diff
@@ -562,6 +561,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OBJC_BRIDGING_HEADER = "";
 				SWIFT_OBJC_INTERFACE_HEADER_NAME = "EarlGreyExampleSwift-Swift.h";
+				SWIFT_VERSION = "";
```

Comment:
It used to have a Swift file, which is why it was set to 3.0. Without this, it'll inherit from the parent project which has 3.0, which will trigger the Swift File Bundling in configure_earlgrey..., which will cause the following error:

```
RuntimeError - Bundled swift missing
/Users/tirodkar/.rvm/gems/ruby-2.3.1/gems/earlgrey-0.2.0/lib/earlgrey/configure_earlgrey.rb:382:in `copy_swift_files'
```